client/src/components/Tabs.jsx

```diff
@@ -1,39 +1,19 @@
 import React from "react";
 import { Link, NavLink } from "react-router-dom";
-import { FaLaptop, FaShieldAlt, FaInfoCircle } from "react-icons/fa";
 import "./Tabs.css";
+import { FaLaptop, FaShieldAlt, FaInfoCircle } from "react-icons/fa";
 
-const Tabs = () => {
-  const profileTabs = [
-    {
-      path: "/profile",
-      name: "General",
-      icon: <FaLaptop />,
-    },
-    {
-      path: "/profile/security",
-      name: "Security",
-      icon: <FaShieldAlt />,
-    },
-    {
-      path: "/profile/about",
-      name: "About",
```

Comment:
these will be removed as there is a new folder data, containing data file tabsData.jsx. from there data is passed as props for dynamic rendering based on the page it is called in.